modules/gltf/gltf_document.cpp

```diff
@@ -2882,7 +2887,6 @@ Error GLTFDocument::_parse_meshes(Ref<GLTFState> state) {
 						if (a.has("JOINTS_0") && a.has("JOINTS_1")) {
 							st->set_skin_weight_count(SurfaceTool::SKIN_8_WEIGHTS);
 						}
-						st->deindex();
```

Comment:
Not an action item, but can I ask what deindex() was for before, and why it is no longer desired?

Reply:
Revert to the old way.

Reply:
Current code always indexes.